### PR TITLE
chore(renovate): only auto rebase renovate PRs when conflicted

### PR DESCRIPTION
**Pull request by bkrem**
## Context

- Renovatebot will by default auto rebase all open renovate PRs when the base branch is updated.
- This is causing Vercel build cascades that cause long queue times unnecessarily for more important builds.
- We already use this rule here: https://github.com/WalletConnect/web-examples/blob/main/renovate.json#L10

### CODE REVIEW

**Review by chris13524 — COMMENTED** [review]
I don't think this works because PRs must incorporate all changes before they can be merged, not just when conflicted

**Comment by bkrem** [comment]
> I don't think this works because PRs must incorporate all changes before they can be merged, not just when conflicted

Point is that you manually trigger a rebase via the checkbox in the renovate PR when necessary:
Screenshot 2024-06-12 at 16 34 32

This isn't a must have but it was pretty painful just now when we were trying to deploy the website changes and renovatebot kept clogging up the vercel queue with rebases of open PRs

**Review by chris13524 — APPROVED** [review]

**Review by tomiir — APPROVED** [review]

**Pull request merged**
